kola/register/register.go

```diff
@@ -49,6 +49,10 @@ type Test struct {
 	Architectures    []string // whitelist of machine architectures supported -- defaults to all
 	Flags            []Flag   // special-case options for this test
 
+	// FailFast skips any sub-test that occurs after a sub-test has
+	// failed.
+	FailFast bool
```

Comment:
More descriptive name?

Reply:
Any ideas? This sort of functionality in other test runners (including [golang](https://go-review.googlesource.com/c/go/+/74450)) usually uses `FailFast` or some derivative.

Reply:
Ah, if it's standard terminology let's stick with it.